parts/k8s/addons/kubernetesmasteraddons-azure-cloud-provider-deployment.yaml

```diff
@@ -48,8 +48,35 @@ metadata:
 roleRef:
   kind: ClusterRole
   apiGroup: rbac.authorization.k8s.io
-  name: system:azure-persistent-volume-binder 
+  name: system:azure-persistent-volume-binder
 subjects:
 - kind: ServiceAccount
   name: persistent-volume-binder
   namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Are these additional `ClusterRole` and `ClusterRoleBinding` definitions compatible w/ < 1.15 k8s?

Reply:
I posed that question to @feiskyer who introduced this into upstream k8s. Depending on the answer we may want to move this modified spec into `parts/k8s/addons/1.15/`

Reply:
yep, they're compatible with old versions, moving to 1.15 dir also works